rainbows.rb

```diff
@@ -27,3 +30,43 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+rainbow = [
+  red: {
+    r: 255,
+    g: 0,
+    b: 0
+  },
+  orange: {
+    r: 255,
+    g: 127,
+    b: 0
+  },
+  yellow: {
+    r: 255,
+    g: 255,
+    b: 0
+  },
+  green: {
+    r: 0,
+    g: 127,
+    b: 0
+  },
+  blue: {
+    r: 0,
+    g: 0,
+    b: 255
+  },
+  indigo: {
+    r: 75,
+    g: 0,
+    b: 130
+  },
+  violet: {
+    r: 143,
+    g: 0,
+    b: 255
+  }
+]
+
+puts rainbow.map { |color_hash|  color_hash.keys  }
```

Comment:
This solution looks like it works, but it does not actually create an Array of Hashes as was asked in the problem. We'll go over in class tomorrow.